vulture/core.py

```diff
@@ -232,14 +243,14 @@ def handle_syntax_error(e):
 
     def scavenge(self, paths, exclude=None):
```

Comment:
You mean, convert `paths` to `Path` objects at the start of the function itself?

Reply:
Exactly. Usually, you want to convert to the internal representation (pathlib) as early as possible and convert back (to strings) as late as possible to avoid having to convert back and forth between the two representations.